apps/fishing-map/features/workspace/workspace.utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Workspace } from '@globalfishingwatch/api-types'
+import { PUBLIC_SUFIX } from 'data/config'
+import { AppWorkspace } from 'features/workspaces-list/workspaces-list.slice'
+import { WorkspaceState } from 'types'
+
+export const getWorkspaceLabel = (workspace: AppWorkspace | Workspace<WorkspaceState, string>) => {
```

Comment:
Can you use the same helper for the [workspace title here](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/features/workspace/Workspace.tsx#L141) ?